shpec.plugin.zsh

```diff
@@ -0,0 +1,15 @@
+##########################################################
+#							 #
+#                _____ __                   		 #
+#               / ___// /_  ____  ___  _____		 #
+#               \__ \/ __ \/ __ \/ _ \/ ___/		 #
+#              ___/ / / / / /_/ /  __/ /__  		 #
+#             /____/_/ /_/ .___/\___/\___/  		 #
+#                       /_/                 		 #
+#        						 #
+##########################################################
+
+
+## just create an alias shpec pointing to where the spec
+## executable really is (in this folder)
+alias shpec="bash $(dirname $0:A)/bin/shpec"
```

Comment:
not really, the way antigen works is that it sources the `myname.plugin.zsh`.
So with a simlink it would source the binary